src/signals/my-incidents/pages/Detail.tsx

```diff
@@ -56,19 +67,16 @@ export const Detail = () => {
         >
           <title>{`Meldingsnummer: ${incidentDisplay.current}`}</title>
         </Helmet>
-        {showMap && incidentsDetail?.location ? (
-          <Map
-            close={() => setShowMap(false)}
-            location={incidentsDetail.location}
-          />
+        {showMap && data?.location ? (
```

Comment:
Location is not optional. So changing the types makes this obsolete. Also, one level down (in IncidentDetailComponent there is an early return that checks if data exists. Might be better to add the early return in this component since you do not want to return anything if you don't have the info.

Reply:
yep!